demos/src/baz.rs

```diff
@@ -3,15 +3,22 @@
 use metered::{metered, ErrorCount, HitCount, InFlight, ResponseTime};
 use thiserror::Error;
 
-#[metered::error_count(name = BazErrorCount)]
+#[metered::error_count(name = LibErrorCount)]
 #[derive(Debug, Error)]
-pub enum BazError {
+pub enum LibError {
     #[error("I failed!")]
     Failure,
     #[error("Bad input")]
     BadInput,
 }
 
+#[metered::error_count(name = BazErrorCount)]
+#[derive(Debug, Error)]
+pub enum BazError {
+    #[error("lib error: {0}")]
+    Lib(#[from] #[nested(LibErrorCount)] LibError),
```

Comment:
Do we need to specify `LibErrorCount` here?  Couldn't  `error_count` implement a trait that returns the associated registry type?

Reply:
That's a good idea, all done.